watcher/src/watcher_db.rs

```diff
@@ -308,4 +348,53 @@ mod test {
 
         assert_eq!(watcher_db.get_block_signatures(1).unwrap().len(), 2);
     }
+
+    // Highest synced block should return the minimum highest synced block for all URLs
+    #[test_with_logger]
+    fn test_highest_synced(logger: Logger) {
+        let mut rng: Hc128Rng = Hc128Rng::from_seed([8u8; 32]);
```

Comment:
I think it's generally better to use one of these 2, if possible: https://github.com/mobilecoinofficial/mobilecoin/blob/6f6fcc6badc459e477f0fb38c4a63cae7e27491f/util/test-helper/src/lib.rs#L25-L33

Reply:
Used run_with_one_seed - thanks!